test/test_configuration.rb

```diff
@@ -28,4 +28,15 @@ def test_mising_api_key
       TaxCloud.client.request :dummy, body: {}
     end
   end
+
+  def test_timeout_options
```

Comment:
If this test tests timeout options, then it shouldn't be setting `api_login_id`. I suggest splitting this into `test_open_timeout` and `test_read_timeout`. It would also be good to reach into an actual request and make sure the timeout has been applied at the right place.